inst/stan/wwinference.stan

```diff
@@ -168,10 +169,12 @@ transformed parameters {
   vector[ot + uot + ht] state_inf_per_capita = rep_vector(0, uot + ot + ht); // state level incident infections per capita
   matrix[n_subpops, ot + ht] model_log_v_ot; // expected observed viral genomes/mL at all observed and forecasted times
   real<lower=0> g = pow(log10_g, 10); // Estimated genomes shed per infected individual
-  real<lower=0> i0 = i0_over_n * state_pop; // Initial absolute infection incidence
-  vector[n_subpops] i0_site_over_n; // site-level initial
-  // per capita infection incidence
-  vector[n_subpops] growth_site;
+  vector<lower=0, upper=1>[n_subpops] i_first_obs_over_n_site = inv_logit(logit(i_first_obs_over_n) +
```

Comment:
Slight slight personal preference when reading stan code that the defining of variables, unless  extremely straightforward, just defines what they are and then the actual transformation happens sequentially.

So instead of this happening here, have it defined here and then have it happening right outside the loop

Reply:
In the past, I've sometimes defined a helper function:

```Stan
vector center(real loc, real scale, vector deviations) {
   return loc + scale * deviations;
}
```
and then
```Stan
  vector[n_subpops] initial_exp_growth_rate_site = center(
      mean_initial_exp_growth_rate,
      sigma_initial_exp_growth_rate,
      eta_initial_exp_growth_rate);
```

Reply:
I think I pretty much only do that if it is just a scalar transformation. Happy to add the function though, but I'd still prefer the operation be called right before the for loop where it is used.